hack/ci/e2e.sh

```diff
@@ -130,6 +133,34 @@ run_tests() {
     KUBECONFIG="$(kind get kubeconfig-path)"
     export KUBECONFIG
 
+    if [[ "${IP_FAMILY:-ipv4}" == "ipv6" ]]; then
```

Comment:
this is not great but not exactly our fault and we can revisit it later and it's orthogonal to the actual CLI work for ipv6